tests/unit/test_zigzag.py

```diff
@@ -9,6 +9,7 @@
 from hashlib import md5
 from zigzag.zigzag import ZigZag
 from swagger_client.rest import ApiException
+from tests.unit.conftest import merge_dicts
```

Comment:
Is the intention to drop support for `pytest.helpers` namespace? Because the `merge_dicts` helper is still decorated to be part of the helpers namespace.